src/t_zset.c

```diff
@@ -456,8 +474,9 @@ static unsigned long zslDeleteRangeByScore(zskiplist *zsl, zrangespec *range, ha
     while (x && zslValueLteMax(x->score, range)) {
         zskiplistNode *next = x->level[0].forward;
         zslDeleteNode(zsl, x, update);
-        hashtableDelete(ht, x->ele);
-        zslFreeNode(x); /* Here is where x->ele is actually released. */
+        sds ele = zslGetNodeElement(x);
+        hashtableDelete(ht, ele);
```

Comment:
nit: although this type has no entry destructor, I would be happy if we could explicitly use hashtablepop instead just to be clear that we are NOT planning to delete the element.